index.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 const {htmlEscape} = require('escape-goat');
 
-module.exports = (template, data) => {
+class MissingValueError extends Error {
+	constructor(key) {
+		super(`Missing a value for ${key ? `the placeholder: ${key}` : 'a placeholder'}`, key);
+		this.name = 'MissingValueError';
+		this.key = key;
+	}
+}
+
+const defaultOptions = {
+	ignoreMissing: false,
+	transform: ({value}) => value
+};
+
+module.exports = (template, data, options = {}) => {
```

Comment:
You can simplify defaults handling:

```suggestion
module.exports = (template, data, {ignoreMissing = false} = {}) => {
```